server.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const fs = require('fs');
 const passport = require('passport');
-const env = process.env.NODE_ENV || 'development';
```

Comment:
change this default back to development.

Reply:
done